packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -75,7 +75,7 @@ export class MeterProvider implements api.MeterProvider {
     if (this._config.exporter) {
       this._config.exporter.shutdown();
     }
-    await Promise.all(
+    return Promise.all(
```

Comment:
this is fine, but you can remove async now from the function `_shutdownAllMeters` as nothing is async anymore here